README.md

```diff
@@ -110,7 +169,9 @@ module "create_account_roles"{
 
 ```
 
-* Least AWS Permissions required to run the terraform
+#### Sample of required permissions
```

Comment:
Do we validate that those permissions are enough?
Need to verify that the CI user has only those permissions
@amalykhi any thoughts?